centreon/www/include/options/media/images/DB-Func.php

```diff
@@ -220,15 +220,18 @@ function moveImg($img_id, $dir_alias)
         return;
     }
     $img_info = $prepare->fetch(PDO::FETCH_ASSOC);
-
+    $image_info_path = basename($img_info["img_path"]);
+    $image_info_dir_alias = basename($img_info["dir_alias"]);
     if ($dir_alias) {
         $dir_alias = sanitizePath($dir_alias);
     } else {
-        $dir_alias = $img_info["dir_alias"];
+        $dir_alias = $image_info_dir_alias;
     }
     if ($dir_alias != $img_info["dir_alias"]) {
-        $oldpath = $mediadir . $img_info["dir_alias"] . "/" . $img_info["img_path"];
-        $newpath = $mediadir . $dir_alias . "/" . $img_info["img_path"];
+
+
+        $oldpath = $mediadir . $image_info_dir_alias . "/" . $image_info_path;
+        $newpath = $mediadir . $dir_alias . "/" . $image_info_path;
```

Comment:
$dir_alias and $image_info_dir_alias are the same variable.

So you could just use $dir_alias, or remote the dir_alias assignment l.228 and use instead $image_info_dir_alias here (and l.230 in condition)

Reply:
I guess in this case we are inside a condition of `($dir_alias != $img_info["dir_alias"]) ` to handle the case of:
a $dir_alias injected in function params wich does not equal $image_info_dir_alias(wich is  imported from db)